uniffi_macros/src/util.rs

```diff
@@ -216,14 +216,33 @@ pub(crate) fn tagged_impl_header(
     }
 }
 
-pub(crate) fn derive_ffi_traits(ty: &Ident, udl_mode: bool) -> TokenStream {
+pub(crate) fn derive_all_ffi_traits(ty: &Ident, udl_mode: bool) -> TokenStream {
```

Comment:
These names kinda bother me a little - `derive_all_ffi_traits` seems to be deriving for a single ident where as `derive_ffi_traits` takes an array of identifiers. It would be great if you could maybe come up with alternatives to make this a little clearer?

Reply:
oh right, doh! Swapping those 2 args sounds great, thanks!